### PR TITLE
Rollup and new folder structure

### PR DESCRIPTION
**Pull request by vkalpias**
Now using rollup instead of webpack, Removed components and binding folders and put everything under root folder.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,12 +75,14 @@
...
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-rename": "^1.0.1",
+    "rollup-plugin-sass": "^1.2.6",
     "sass": "^1.32.12",
     "sass-loader": "^9.0.2",
     "tsd-jsdoc": "^2.5.0",
-    "webpack": "^5.51.1",
-    "webpack-cli": "4.8.0",
-    "@playcanvas/observer": "1.0.4"
+    "webpack": "^5.51.1"
```
Comment (kungfooman) [inline package.json:85]: `npm uninstall webpack` or is it still required for other things?

File: package.json [hunk 2]
```diff
@@ -14,8 +14,7 @@
     "url": "git+https://github.com/playcanvas/pcui.git"
   },
   "scripts": {
-    "build": "webpack --config webpack.config.js && MODULE=true webpack --config webpack.config.js && MODULE=true MODULE_STRIP=true webpack --config webpack.config.js",
-    "watch": "ENVIRONMENT=development webpack --config webpack.config.js --watch",
+    "build": "rm -r dist; rollup -c; rm dist/index.mjs; rm dist/bundle/index.mjs",
```
Comment (kungfooman) [inline package.json:17]: This will be linux centric again, for Windows its `rimraf`

**Review by kungfooman — COMMENTED** [review]
Nice work, glad to see this!

File: package.json (hunk 1 above)
Comment (vkalpias) [reply package.json:86]: we still need it for storybook

File: package.json (hunk 2 above)
Comment (vkalpias) [reply package.json]: Good point changed that to use rollup instead.

**Review by ellthompson — APPROVED** [review]

**Pull request merged**